### PR TITLE
chore: wrap up readme and license

### PR DESCRIPTION
**Pull request by perfectmak**
This create a README for the Checkpoint project and provides relevant
links to the Checkpoint documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+- [How it works](https://docs.checkpoint.fyi/#how-it-works)
+- [Quickstart Guide](https://docs.checkpoint.fyi/guides/quickstart)
+- Core Concepts
+  - [Configuration](https://docs.checkpoint.fyi/core-concepts/checkpoint-configuration)
+  - [Entity Schema](https://docs.checkpoint.fyi/core-concepts/entity-schema)
+  - [Data Writers](https://docs.checkpoint.fyi/core-concepts/checkpoint-writers)
+
+## Contribution Guideline
+
+Join snapshot's [discord](https://discord.gg/snapshot) to further discuss and engage with the team.
```
Comment (bonustrack) [inline README.md:32]:
```suggestion
Join snapshot's [Discord](https://discord.gg/snapshot) to further discuss and engage with the team.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+npm install @snapshot-labs/checkpoint
+```
+
+## Documentation
+
+- [How it works](https://docs.checkpoint.fyi/#how-it-works)
+- [Quickstart Guide](https://docs.checkpoint.fyi/guides/quickstart)
+- Core Concepts
+  - [Configuration](https://docs.checkpoint.fyi/core-concepts/checkpoint-configuration)
+  - [Entity Schema](https://docs.checkpoint.fyi/core-concepts/entity-schema)
```
Comment (bonustrack) [inline README.md:27]:
```suggestion
  - [Entity schema](https://docs.checkpoint.fyi/core-concepts/entity-schema)
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+  - [Data Writers](https://docs.checkpoint.fyi/core-concepts/checkpoint-writers)
```
Comment (bonustrack) [inline README.md:28]:
```suggestion
  - [Data writers](https://docs.checkpoint.fyi/core-concepts/checkpoint-writers)
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+
+## Contribution Guideline
```
Comment (bonustrack) [inline README.md:30]:
```suggestion
## Contribution guideline
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,36 @@
...
+Checkpoint is an NPM package that can be installed through the follow command:
+
+```tsx
+npm install @snapshot-labs/checkpoint
+```
+
+## Documentation
+
+- [How it works](https://docs.checkpoint.fyi/#how-it-works)
+- [Quickstart Guide](https://docs.checkpoint.fyi/guides/quickstart)
```
Comment (bonustrack) [inline README.md:24]:
```suggestion
- [Quickstart guide](https://docs.checkpoint.fyi/guides/quickstart)
```

**Comment by bonustrack** [comment]
Can you use "master" as main branch? Instead of "main". This is what we use on others repo

**Comment by perfectmak** [comment]
> Can you use "master" as main branch? Instead of "main". This is what we use on others repo

Yeah, sure. I'm used to starting out new branches on `main` but having it on `master`so all repos are uniform is fine with me. I'll merge this in and then create a master branch that we can set as the default branch going forward.

**Pull request merged**